server/admin_management/api/app/utils/audit_util.py

```diff
@@ -11,6 +13,7 @@
 class AuditEventType(str, Enum):
     CREATE_APPLICATION_ADMIN_ACCESS = "Grant User Application Admin Access"
     REMOVE_APPLICATION_ADMIN_ACCESS = "Remove User Application Admin Access"
+    CREATE_ACCESS_CONTROL_PRIVILIEGE = "Create Access Control Privilege"
```

Comment:
Since the audience is for security auditor, maybe we can give description as "Grant Delegated Admin Privilege" or similar?